packages/adapter-netlify/package.json

```diff
@@ -26,7 +26,7 @@
 	"scripts": {
 		"dev": "rimraf files && rollup -cw",
 		"build": "rimraf files && rollup -c && cp src/edge.js files/edge.js",
-		"test": "uvu src \"(spec\\.js|test[\\\\/]index\\.js)\"",
+		"test": "vitest run src \"(spec\\.js|test[\\\\/]index\\.js)\"",
```

Comment:
this should be `"test":"vitest run"` and add `"test:watch":"vitest"`  so that you can pass extra args on the cli when eg you do `pnpm test some-test.spec.ts` and it just runs that. move the default includes into a config.

Reply:
the filters were all unnecessary leftovers, so I've removed them (the also don't actually work for me due to a vitest bug https://github.com/vitest-dev/vitest/issues/3344)

goo catch on `uvu` vs `vitest` here. I've fixed that as well